ivn/core.py

```diff
@@ -639,7 +639,9 @@ def __forward(self, src_ip, src_port, dst_port):
 
     @staticmethod
     def build(portforward, logger):
-        rules = portforward["rules"]
+        rules = portforward.get("rules", None)
```

Comment:
suggest to change to:
> rules = portforward.get("rules")
io_interfaces = portforward.get("io_interfaces")

simple and safe because there is a "if rules and io_interface:" later.